### PR TITLE
Adjust issue templates

### PR DESCRIPTION
**Pull request by leventov**
As per [this discussion](https://lists.apache.org/thread.html/e16528f7552826c2e3435b6a4ce4079d0a8f2670b5e0f7204ec81fc8@%3Cdev.druid.apache.org%3E) in the mailing list.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+assignees: ''
+
+---
+
+### Description
+
+Please describe the feature or change with as much detail as possible. 
+
+If you have a detailed implementation in mind and wish to contribute that implementation yourself, and the change that
+you are planning would require a 'Design Review' tag because it intoroduces or changes some APIs, or it is large and
```
Comment (jon-wei) [inline .github/ISSUE_TEMPLATE/feature.md]: "intoroduces" -> "introduces"

Comment (leventov) [reply .github/ISSUE_TEMPLATE/feature.md]: Thanks, fixed

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
 assignees: ''
 
 ---
 
 Please provide a detailed title (e.g. "Broker crashes when using TopN query with Bound filter" instead of just "Broker crashes").
 
-# Affected Version
+### Affected Version
 
 The Druid version where the bug was encountered.
```
Comment (jon-wei) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Let's change `bug` to `'problem` here and elsewhere, LGTM otherwise

Comment (leventov) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Thanks, changed.

**Pull request merged**